scalameta/scalameta/shared/src/test/scala/scala/meta/tests/parsers/ParseSuite.scala

```diff
@@ -12,6 +12,14 @@ import org.scalameta.logger
 class ParseSuite extends FunSuite with CommonTrees {
   val EOL = scala.compat.Platform.EOL
   val escapedEOL = if (EOL == "\n") """\n""" else """\r\n"""
+
+
+  def assertSameLines(actual: String, expected: String) = {
```

Comment:
Maybe add a note saying this should eventually be replaced with `DiffAssertions.assertNoDiff`, which is available in scalameta-testkit. It works similarly but produces nice unified diffs on failures. I'm planning to move `scalametaJVM/test` to `testsJVM`, but haven't gotten around to it yet.